### PR TITLE
Cleanup docs

### PR DESCRIPTION
**Pull request by SegueII**

### CODE REVIEW

File: docs/zh/daemon/intro.md [hunk 1]
```diff
@@ -47,8 +45,8 @@ pri_validator.json是[Tendermint Key](../concepts/validator-faq.md#tendermint-
 
 ### config.toml
 
-config.toml是节点的非共识配置。不同的节点可以根据自己的情况进行配置。常见的修改是`persistent_peers` /`moniker` /`laddr`
+config.toml是节点的非共识配置。不同的节点可以根据自己的情况进行配置。常见的修改是`persistent_peers`、`moniker`、`laddr`
 
 ### iris.toml
```
Comment (chengwenxi) [inline docs/zh/daemon/intro.md:50]: `iris.toml` has been removed.

Comment (chengwenxi) [reply docs/zh/daemon/intro.md:50]: we should add some description for `app.toml`

**Review by chengwenxi — APPROVED** [review]

File: docs/concepts/validator-faq.md [hunk 2]
```diff
@@ -249,4 +249,4 @@ That means you have **Lost Your Validator Forever!** You can only create a new o
 ## Community Channels
 
 - Riot chat: <https://riot.im/app/#/room/#irisvalidators:matrix.org>
-- IRIShub Validator Working Group QQ Group：834063323
+- IRIShub Validator Working Group QQ Group: 834063323
```
Comment (dreamer-zq) [inline docs/concepts/validator-faq.md:252]: add `discord` channel ?

File: docs/cli-client/service.md [hunk 3]
```diff
@@ -296,17 +296,17 @@ iris tx service call [flags]
...
+| ----------------- | ------- | ---------------------------------------------------------------------------------------------------------------------- | -------- |
+| --service-name    |         | Service name                                                                                                           | Yes      |
+| --providers       |         | Provider list to request                                                                                               | Yes      |
+| --service-fee-cap |         | Maximum service fee to pay for a single request                                                                        | Yes      |
+| --data            |         | Content or file path of the request input, which is an Input JSON Schema instance                                      | Yes      |
+| --timeout         |         | Request timeout                                                                                                        | Yes      |
+| --super-mode      | false   | Indicate if the signer is a super user                                                                                 |          |
```
Comment (haifengxi) [inline docs/cli-client/service.md:306]: remove `super-mode` everywhere

File: docs/endpoints/intro.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 # Introduction
 
-This document presents an overview of all the endpoints and IRISHub-SDK a node exposes.
+This document presents an overview of all the endpoints and IRIShub-SDK a node exposes.
```
Comment (haifengxi) [inline docs/endpoints/intro.md:3]: This document presents an overview of all the API endpoints a node exposes and the higher-level client SDKs.

File: docs/endpoints/sdk.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
-# IRISHub SDKs
+# IRIShub SDKs
```
Comment (haifengxi) [inline docs/endpoints/sdk.md:1]: IRIShub Client SDKs

**Review by haifengxi — CHANGES_REQUESTED** [review]

File: docs/concepts/validator-faq.md (hunk 2 above)
Comment (SegueII) [reply docs/concepts/validator-faq.md:252]: I think it is better to maintain all contact information in another PR

**Review by haifengxi — APPROVED** [review]

**Pull request merged**